x/staking/keeper/querier.go

```diff
@@ -57,7 +57,7 @@ func queryValidators(ctx sdk.Context, req abci.RequestQuery, k Keeper) ([]byte,
 	}
 
 	validators := k.GetAllValidators(ctx)
-	filteredVals := make([]types.Validator, 0, len(validators))
```

Comment:
undo. This should never return nil to begin with.

Reply:
Done :+1: